optional_plugins/spawner_remote/README.rst

```diff
@@ -0,0 +1,37 @@
+=====================
+Remote Spawner Plugin
+=====================
+
+This plugin makes use of remote aexpect sessions (to remote hosts or
```

Comment:
Can you please add a link to aexpect repository, users might not be aware of it and a link could help here.